### ISSUE TITLE
Rendering artifacts for superposed PointBuffers

### ISSUE THREAD

**Issue by dominiquesydow** [nglviewer/ngl#868]
Hi,

Thanks a lot for your work on NGL!

I would like to report a potential bug. 
In one of our projects, we use `PointBuffer` to visualize multiple point clouds (each cloud represented by one `PointBuffer`). 

---

Implementing this using the Python package `nglview`, we see rendering artifacts when two different `PointBuffer` are superposed:
![image](https://user-images.githubusercontent.com/7207093/119619480-53cca800-be04-11eb-8ccb-45b87e284696.png)
(PR https://github.com/dominiquesydow/dynophores/pull/28)

---

Please see a minimum example on [CodePen](https://codepen.io/dominiquesydow/pen/VwpvWQE) using `NGL` directly:
![image](https://user-images.githubusercontent.com/7207093/119619660-84acdd00-be04-11eb-8517-40cfdb94f587.png)

---

We do have an alternative visualization using `SphereBuffer`. But in case someone has capacity to look into this or ideas why this is happening, we are happy to hear about it.

Thank you!

**Comment by dominiquesydow** [comment]
Thank you for getting back to me so quickly!

Could you please check out the example on CodePen I posted above and let me know now if this counts as minimum example? This is where I generated the second figure from:
https://codepen.io/dominiquesydow/pen/VwpvWQE

If that example has e.g. too many points I will prepare something smaller.

I think I only see this artifact if I view the blue points on top of the red points.

**Comment by fredludlow** [comment]
🤦‍♂️ Sorry, didn't see the codepen link...

**Comment by dominiquesydow** [comment]
Hi @fredludlow, were you able to reproduce the rendering issue?

**Comment by fredludlow** [comment]
Yes, I can reproduce it, but I suspect it's related to other issues around blending semi-transparent objects (and not an easy fix) - see #552 and #592, #598, #852 etc :(

**Comment by dominiquesydow** [comment]
Oh I see, thank you for the update!
